### PR TITLE
[DROOLS-6450] Implement NormContinuous

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @kostola  @jiripetrlik

See https://issues.redhat.com/browse/DROOLS-6450

### CODE REVIEW

File: kie-pmml-trusty/kie-pmml-api/src/main/java/org/kie/pmml/api/enums/OUTLIER_TREATMENT_METHOD.java [hunk 1]
```diff
@@ -23,15 +23,11 @@
...
+public enum OUTLIER_TREATMENT_METHOD implements Named {
 
-    ASSOCIATION_RULES("associationRules"),
-    SEQUENCES("sequences"),
-    CLASSIFICATION("classification"),
-    REGRESSION("regression"),
-    CLUSTERING("clustering"),
-    TIME_SERIES("timeSeries"),
-    MIXED("mixed");
+    AS_IS("asIs"),
```
Comment (jiripetrlik) [inline kie-pmml-trusty/kie-pmml-api/src/main/java/org/kie/pmml/api/enums/OUTLIER_TREATMENT_METHOD.java:28]: What is the reason for removing values from this enum?

**Review by jiripetrlik — APPROVED** [review]
LGTM. I have one question.

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-tree/kie-pmml-models-tree-model/src/main/resources/KiePMMLTreeModelTemplate.tmpl [hunk 2]
```diff
@@ -4,14 +4,8 @@ import java.util.Arrays;
...
-import org.kie.pmml.commons.model.expressions.KiePMMLApply;
-import org.kie.pmml.commons.model.expressions.KiePMMLConstant;
-import org.kie.pmml.commons.model.expressions.KiePMMLFieldRef;
-import org.kie.pmml.commons.transformations.KiePMMLDefineFunction;
-import org.kie.pmml.commons.transformations.KiePMMLDerivedField;
-import org.kie.pmml.commons.transformations.KiePMMLLocalTransformations;
-import org.kie.pmml.commons.transformations.KiePMMLParameterField;
-import org.kie.pmml.commons.transformations.KiePMMLTransformationDictionary;
+import org.kie.pmml.commons.model.expressions.*;
+import org.kie.pmml.commons.transformations.*;
```
Comment (danielezonca) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-tree/kie-pmml-models-tree-model/src/main/resources/KiePMMLTreeModelTemplate.tmpl:8]: Is this change on purpose?

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-scorecard/kie-pmml-models-scorecard-model/src/main/resources/KiePMMLScorecardModelTemplate.tmpl [hunk 3]
```diff
@@ -8,14 +8,8 @@ import org.kie.pmml.api.enums.PMML_MODEL;
...
-import org.kie.pmml.commons.model.expressions.KiePMMLApply;
-import org.kie.pmml.commons.model.expressions.KiePMMLConstant;
-import org.kie.pmml.commons.model.expressions.KiePMMLFieldRef;
-import org.kie.pmml.commons.transformations.KiePMMLDefineFunction;
-import org.kie.pmml.commons.transformations.KiePMMLDerivedField;
-import org.kie.pmml.commons.transformations.KiePMMLLocalTransformations;
-import org.kie.pmml.commons.transformations.KiePMMLParameterField;
-import org.kie.pmml.commons.transformations.KiePMMLTransformationDictionary;
+import org.kie.pmml.commons.model.expressions.*;
+import org.kie.pmml.commons.transformations.*;
```
Comment (danielezonca) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-scorecard/kie-pmml-models-scorecard-model/src/main/resources/KiePMMLScorecardModelTemplate.tmpl:12]: Same here

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-regression/kie-pmml-models-regression-model/src/main/resources/KiePMMLRegressionModelTemplate.tmpl [hunk 4]
```diff
@@ -4,14 +4,8 @@ import java.util.Arrays;
...
-import org.kie.pmml.commons.model.expressions.KiePMMLApply;
-import org.kie.pmml.commons.model.expressions.KiePMMLConstant;
-import org.kie.pmml.commons.model.expressions.KiePMMLFieldRef;
-import org.kie.pmml.commons.transformations.KiePMMLDefineFunction;
-import org.kie.pmml.commons.transformations.KiePMMLDerivedField;
-import org.kie.pmml.commons.transformations.KiePMMLLocalTransformations;
-import org.kie.pmml.commons.transformations.KiePMMLParameterField;
-import org.kie.pmml.commons.transformations.KiePMMLTransformationDictionary;
+import org.kie.pmml.commons.model.expressions.*;
+import org.kie.pmml.commons.transformations.*;
```
Comment (danielezonca) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-regression/kie-pmml-models-regression-model/src/main/resources/KiePMMLRegressionModelTemplate.tmpl:8]: Same here

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-mining/kie-pmml-models-mining-model/src/main/resources/KiePMMLSegmentTemplate.tmpl [hunk 5]
```diff
@@ -3,12 +3,7 @@ package org.kie.pmml.models.mining.model.segmentation;
 import java.util.Arrays;
 import java.util.Collections;
 
-import org.kie.pmml.commons.model.predicates.KiePMMLCompoundPredicate;
-import org.kie.pmml.commons.model.predicates.KiePMMLFalsePredicate;
-import org.kie.pmml.commons.model.predicates.KiePMMLPredicate;
-import org.kie.pmml.commons.model.predicates.KiePMMLSimplePredicate;
-import org.kie.pmml.commons.model.predicates.KiePMMLSimpleSetPredicate;
-import org.kie.pmml.commons.model.predicates.KiePMMLTruePredicate;
+import org.kie.pmml.commons.model.predicates.*;
```
Comment (danielezonca) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-mining/kie-pmml-models-mining-model/src/main/resources/KiePMMLSegmentTemplate.tmpl:6]: Same here

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-mining/kie-pmml-models-mining-model/src/main/resources/KiePMMLMiningModelTemplate.tmpl [hunk 6]
```diff
@@ -4,14 +4,8 @@ import java.util.Arrays;
...
-import org.kie.pmml.commons.model.expressions.KiePMMLApply;
-import org.kie.pmml.commons.model.expressions.KiePMMLConstant;
-import org.kie.pmml.commons.model.expressions.KiePMMLFieldRef;
-import org.kie.pmml.commons.transformations.KiePMMLDefineFunction;
-import org.kie.pmml.commons.transformations.KiePMMLDerivedField;
-import org.kie.pmml.commons.transformations.KiePMMLLocalTransformations;
-import org.kie.pmml.commons.transformations.KiePMMLParameterField;
-import org.kie.pmml.commons.transformations.KiePMMLTransformationDictionary;
+import org.kie.pmml.commons.model.expressions.*;
+import org.kie.pmml.commons.transformations.*;
```
Comment (danielezonca) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-mining/kie-pmml-models-mining-model/src/main/resources/KiePMMLMiningModelTemplate.tmpl:8]: Same here

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-tree/kie-pmml-models-drools-tree-model/src/main/resources/KiePMMLTreeModelTemplate.tmpl [hunk 7]
```diff
@@ -4,14 +4,8 @@ import java.util.Arrays;
...
-import org.kie.pmml.commons.model.expressions.KiePMMLApply;
-import org.kie.pmml.commons.model.expressions.KiePMMLConstant;
-import org.kie.pmml.commons.model.expressions.KiePMMLFieldRef;
-import org.kie.pmml.commons.transformations.KiePMMLDefineFunction;
-import org.kie.pmml.commons.transformations.KiePMMLDerivedField;
-import org.kie.pmml.commons.transformations.KiePMMLLocalTransformations;
-import org.kie.pmml.commons.transformations.KiePMMLParameterField;
-import org.kie.pmml.commons.transformations.KiePMMLTransformationDictionary;
+import org.kie.pmml.commons.model.expressions.*;
+import org.kie.pmml.commons.transformations.*;
```
Comment (danielezonca) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-tree/kie-pmml-models-drools-tree-model/src/main/resources/KiePMMLTreeModelTemplate.tmpl:8]: Same here

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-scorecard/kie-pmml-models-drools-scorecard-model/src/main/resources/KiePMMLScorecardModelTemplate.tmpl [hunk 8]
```diff
@@ -4,14 +4,8 @@ import java.util.Arrays;
...
-import org.kie.pmml.commons.model.expressions.KiePMMLApply;
-import org.kie.pmml.commons.model.expressions.KiePMMLConstant;
-import org.kie.pmml.commons.model.expressions.KiePMMLFieldRef;
-import org.kie.pmml.commons.transformations.KiePMMLDefineFunction;
-import org.kie.pmml.commons.transformations.KiePMMLDerivedField;
-import org.kie.pmml.commons.transformations.KiePMMLLocalTransformations;
-import org.kie.pmml.commons.transformations.KiePMMLParameterField;
-import org.kie.pmml.commons.transformations.KiePMMLTransformationDictionary;
+import org.kie.pmml.commons.model.expressions.*;
+import org.kie.pmml.commons.transformations.*;
```
Comment (danielezonca) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-scorecard/kie-pmml-models-drools-scorecard-model/src/main/resources/KiePMMLScorecardModelTemplate.tmpl:8]: Same here

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-clustering/kie-pmml-models-clustering-model/src/main/resources/KiePMMLClusteringModelTemplate.tmpl [hunk 9]
```diff
@@ -4,14 +4,8 @@ import java.util.Arrays;
...
-import org.kie.pmml.commons.model.expressions.KiePMMLApply;
-import org.kie.pmml.commons.model.expressions.KiePMMLConstant;
-import org.kie.pmml.commons.model.expressions.KiePMMLFieldRef;
-import org.kie.pmml.commons.transformations.KiePMMLDefineFunction;
-import org.kie.pmml.commons.transformations.KiePMMLDerivedField;
-import org.kie.pmml.commons.transformations.KiePMMLLocalTransformations;
-import org.kie.pmml.commons.transformations.KiePMMLParameterField;
-import org.kie.pmml.commons.transformations.KiePMMLTransformationDictionary;
+import org.kie.pmml.commons.model.expressions.*;
+import org.kie.pmml.commons.transformations.*;
```
Comment (danielezonca) [inline kie-pmml-trusty/kie-pmml-models/kie-pmml-models-clustering/kie-pmml-models-clustering-model/src/main/resources/KiePMMLClusteringModelTemplate.tmpl:8]: Same here

File: kie-pmml-trusty/kie-pmml-api/src/main/java/org/kie/pmml/api/enums/OUTLIER_TREATMENT_METHOD.java (hunk 1 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-api/src/main/java/org/kie/pmml/api/enums/OUTLIER_TREATMENT_METHOD.java:28]: Because they where not part of OutlierTreatmentMethod, and it was unused before. Probably a wrong copy'n'paste

**Review by danielezonca — APPROVED** [review]
LGTM 👍

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-tree/kie-pmml-models-tree-model/src/main/resources/KiePMMLTreeModelTemplate.tmpl (hunk 2 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-models/kie-pmml-models-tree/kie-pmml-models-tree-model/src/main/resources/KiePMMLTreeModelTemplate.tmpl:8]: @danielezonca 
yes

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-scorecard/kie-pmml-models-scorecard-model/src/main/resources/KiePMMLScorecardModelTemplate.tmpl (hunk 3 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-models/kie-pmml-models-scorecard/kie-pmml-models-scorecard-model/src/main/resources/KiePMMLScorecardModelTemplate.tmpl:12]: @danielezonca 
yes

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-regression/kie-pmml-models-regression-model/src/main/resources/KiePMMLRegressionModelTemplate.tmpl (hunk 4 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-models/kie-pmml-models-regression/kie-pmml-models-regression-model/src/main/resources/KiePMMLRegressionModelTemplate.tmpl:8]: @danielezonca 
yes

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-mining/kie-pmml-models-mining-model/src/main/resources/KiePMMLSegmentTemplate.tmpl (hunk 5 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-models/kie-pmml-models-mining/kie-pmml-models-mining-model/src/main/resources/KiePMMLSegmentTemplate.tmpl:6]: @danielezonca 
yes

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-mining/kie-pmml-models-mining-model/src/main/resources/KiePMMLMiningModelTemplate.tmpl (hunk 6 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-models/kie-pmml-models-mining/kie-pmml-models-mining-model/src/main/resources/KiePMMLMiningModelTemplate.tmpl:8]: @danielezonca 
yes

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-tree/kie-pmml-models-drools-tree-model/src/main/resources/KiePMMLTreeModelTemplate.tmpl (hunk 7 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-tree/kie-pmml-models-drools-tree-model/src/main/resources/KiePMMLTreeModelTemplate.tmpl:8]: @danielezonca 
same here

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-scorecard/kie-pmml-models-drools-scorecard-model/src/main/resources/KiePMMLScorecardModelTemplate.tmpl (hunk 8 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-models/kie-pmml-models-drools/kie-pmml-models-drools-scorecard/kie-pmml-models-drools-scorecard-model/src/main/resources/KiePMMLScorecardModelTemplate.tmpl:8]: @danielezonca 
same here

File: kie-pmml-trusty/kie-pmml-models/kie-pmml-models-clustering/kie-pmml-models-clustering-model/src/main/resources/KiePMMLClusteringModelTemplate.tmpl (hunk 9 above)
Comment (gitgabrio) [reply kie-pmml-trusty/kie-pmml-models/kie-pmml-models-clustering/kie-pmml-models-clustering-model/src/main/resources/KiePMMLClusteringModelTemplate.tmpl:8]: @danielezonca 
same here

**Comment by gitgabrio** [comment]
Jenkins run cdb
